apps/dashboard/src/components/activity/activity-panel.tsx

```diff
@@ -77,7 +103,10 @@ export function ActivityPanel({
               ctaClassName="text-foreground-950"
               variant={'tip'}
               ctaLabel="View Execution"
-              onCtaClick={() => {
+              onCtaClick={(e) => {
+                e.stopPropagation();
+                e.preventDefault();
```

Comment:
don't propagate click event

Reply:
@LetItRock I've faced this issue already on other multiple product areas. Wdyt if we change the primitive Button component to have type="button", so it won't be treated as type submit when wrapped in a form?